common/types.go

```diff
@@ -339,6 +339,16 @@ func (a Address) Value() (driver.Value, error) {
 	return a[:], nil
 }
 
+// CopyPointer returns a new pointer to a copy of the address or nil
+// if [a] is nil.
+func (a *Address) CopyPointer() *Address {
```

Comment:
I think I'd prefer it called `Clone` personally. And the doc could be a bit clearer. 
" Clone returns a pointer to a copy of a (or nil if a is nil)"

Just a suggestion, really

Reply:
`Clone` sounds much better to me. I'll amend this today.